manifests/a/Agilent/LabAdvisor/2.19.503/Agilent.LabAdvisor.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.0.6 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Agilent.LabAdvisor
+PackageVersion: 2.19.503
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+Installers:
+- Architecture: x64
+  InstallerUrl: https://labadvisor.pl29.agilent.com/AgilentLabAdvisor_2.19_SR1.503.msi
+  InstallerSha256: 8576c67bc4189fb727035fb3ff87a7978c67c5bc651aab7a2f8aafa4d5e0f222
+  ProductCode: '{B64A00F5-D3FF-4683-AAB7-84B62EE8FBF1}'
```

Comment:
```suggestion
  ProductCode: '{631D82BE-8FB8-4B7C-9E61-B7DFFBA30601}'
```

Reply:
Yes that´s true, i should have used the script :(